precompiles/utils/src/lib.rs

```diff
@@ -93,7 +93,7 @@ impl<'a> InputReader<'a> {
 
 		self.cursor += 32;
 
-		Ok(H160::from_slice(&data[12..32]))
+		Ok(H160::from_slice(&data[12..32]).into())
```

Comment:
CC @nanocryk @girazoki , I prefer this because then we don't have `.into()` in each precompile

Reply:
also added `read_u32` and `read_balance<Balance: TryFrom<U256>>`